app/nexus/internal/env/backend.go

```diff
@@ -33,7 +37,7 @@ const (
 // case-insensitive.
 //
 // Valid values are:
-//   - "s3": Uses an AWS S3-compatible medium as a backing store.
```

Comment:
No need for an "s3" store type, since we use "lite" in lieu of it.

If we get a request from the community, we can implement one.

Reply:
I think we will eventually want to implement it. but for now, we can remove it.